packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.tsx

```diff
@@ -31,6 +53,11 @@ export class AtomicCommerceProductList
   @InitializeBindings() public bindings!: CommerceBindings;
   public productListing!: ProductListing;
   public search!: Search;
+  private loadingFlag = randomID('firstResultLoaded-');
```

Comment:
Can we change to `firstProductLoaded-` ? 🤔

Reply:
Yes!